src-built-in/components/toolbar/stores/toolbarStore.js

```diff
@@ -239,7 +239,7 @@ class _ToolbarStore {
 			});
 			FSBL.Clients.HotkeyClient.addGlobalHotkey([keys.ctrl, keys.alt, keys.f], (err) => {
 				if (err) { FSBL.Clients.Logger.error(`HotkeyClient.addGlobalHotkey failed, error:`, err); }
-				self.Store.setValue({ field: "searchActive", value: true });
+				this.bringToolbarToFront(true);
```

Comment:
I think we need to have both these lines. The test doesn't indicate so, but I think the test is incomplete.

Reply:
Actually, I seem to be mistaken. I tested it, and everything works fine.